src/main/java/gregtech/api/metatileentity/multiblock/FuelMultiblockController.java

```diff
@@ -63,6 +64,10 @@ protected void addDisplayText(List<ITextComponent> textList) {
                 textList.add(new TextComponentTranslation("gregtech.multiblock.running"));
                 int currentProgress = (int) (recipeMapWorkable.getProgressPercent() * 100);
                 textList.add(new TextComponentTranslation("gregtech.multiblock.progress", currentProgress));
+            } else if (recipeMapWorkable.getPreviousRecipe() != null) {
+                Recipe recipe = recipeMapWorkable.getPreviousRecipe();
+                // add check for not enough fluid or item inputs
```

Comment:
Was more intended to be done here?

Reply:
I was debating on adding a message for if there was insufficient fuel ("Not Enough Fuel" or something) instead of showing the exact L/t required to run the recipe. This comment is where I think that logic would go, should I decide to add the status check message.

Reply:
i've decided to remove this instead, for now